NPS.ID.PublicApi.Models/Properties/AssemblyInfo.cs

```diff
@@ -16,7 +16,7 @@
  #endif
 [assembly: AssemblyCompany("Nord Pool")]
 [assembly: AssemblyProduct("Intraday API")]
-[assembly: AssemblyCopyright("Copyright � 2017")]
+[assembly: AssemblyCopyright("Copyright � 2017")]
```

Comment:
Not a big thing, but it shows to me copyright symbol not correct
![image](https://user-images.githubusercontent.com/19488469/231768879-469642d3-9ad0-43ae-b6cb-514e1600a9ed.png)